src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
+import Categories from './components/categories';
+import NavBar from './components/navBar';
+import Books from './components/books';
+
+function App() {
+  return (
+    <>
+      <Router>
+        <NavBar />
+        <Routes>
+          <Route path="/books" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+          <Route path="/*" element={<Books />} />
```

Comment:
Nice 😎 In order to not show a blank page when visiting the site for the first time, the BookList component is set as the default page